xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -6774,6 +6777,19 @@ glusterd_brick_start(glusterd_volinfo_t *volinfo,
                    "Using %s as sockfile for brick %s of volume %s ",
                    socketpath, brickinfo->path, volinfo->volname);
 
+            /*
+             * Condition to check if the brick socket file is present
+             * in the stated path or not. This helps in preventing
+             * constant re-connect triggered in the RPC layer and also
+             * a log message would help out the user.
+             */
+            ret = sys_lstat(socketpath, &stbuf);
```

Comment:
I think you can call sys_access instead of calling sys_lstat here as we are doing same for pidfile at line no. 6751

Reply:
Ack..I'll make the changes accordingly..